### DISCUSSION TITLE
700 🌟 and Jon as the newest core dev!

### DISCUSSION THREAD

**Discussion by patrick91** [strawberry-graphql/strawberry#510] [Show and tell]
Hello everyone! We just reached 700 stars ✨, but mostly importantly I'm supper happy to introduce @jkimbo as the newest core dev for Strawberry! Sorry it took me a while to write this!

Jon has been previously maintaining graphene-django and has been contributing to Strawberry in the past few months quite a bit, both with code, by highlighting issues and finding rough edges cases (like incompatibilities with MyPy for example) all while trying the library for one of his projects. He’s also helping alongside with @BryceBeagle shaping up the public and internal API of strawberry. 

I’m really happy to have Jon as a core dev! Can’t wait to see your next contributions and all the learnings you’re going to bring from your project!

![](https://media.giphy.com/media/idYdfZ9xxVgVPNXjs0/source.gif)

**Comment by ulgens** [comment]
@jkimbo Congrats dude! :rocket:

**Comment by jkimbo** [comment]
Thanks @patrick91 😊
